pkgs/development/libraries/mesa/default.nix

```diff
@@ -1,10 +1,9 @@
 { stdenv, fetchurl, lib
-, pkgconfig, intltool, autoreconfHook
-, file, expat, libdrm, xorg, wayland, wayland-protocols, openssl
+, pkgconfig, intltool, ninja, meson
+, file, flex, bison, expat, libdrm, xorg, wayland, wayland-protocols, openssl
 , llvmPackages, libffi, libomxil-bellagio, libva-minimal
-, libelf, libvdpau, valgrind-light, python2, python2Packages
+, libelf, libvdpau, valgrind-light, python3, python3Packages
```

Comment:
Use either `python3` with `python3.pkgs` or `python3Packages` with `python3Packages.python`. Mixing the two approaches can lead to conflicts.